### PR TITLE
Fixed ajax

### PR DESCRIPTION
**Pull request by Zjohn64**
Draggable and droppable is now working with a successful http request from the AJAX call

### CODE REVIEW

File: app/views/games/show.html.erb [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
             <% 8.times.each do |col| %>
               <% piece = @game.pieces.where(position_x: col, position_y: row).first %>
               <% if piece.present? %>
-                <td class="present" data-piece_id="<%= piece %>" data-piece_name="<%= piece.color %>_<%= piece.type %>">
-                  <% image = "#{piece.color}_#{piece.type}.png" %>
-                  <%= image_tag image %>
+                <td data-piece_id="<%= piece %>" data-piece_name="<%= piece.color %>_<%= piece.type %>">
+                  <% image ="#{piece.color}_#{piece.type}.png" %>
+                  <%= image_tag(image, class: "piece_image") %>
                 </td>
```
Comment (ulalu) [inline app/views/games/show.html.erb:17]: good refactor!

File: app/assets/javascripts/game_moves.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  $( ".piece_image" ).draggable();
+  $( "td" ).droppable({
+    
+    drop: function( event, ui ) {
+     var game_id = $('#board').data('game_id');
+
+      $.ajax( {  type: 'PUT',
+        url: '/games/' + game_id + '/move',
+        dataType: 'json',
+
```
Comment (ulalu) [inline app/assets/javascripts/game_moves.js:11]: oh nice, a record of the moves! this'll be great if we want to put move logs in the ui anywhere.

File: app/assets/javascripts/game_moves.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        data: {
+          position_x: $(event.target).data('row'),
+          position_y: $(event.target).data('col'),
+          piece_id: $(ui.draggable).data("piece_id")
+        }
+      })
+      .success(function(data){
+       $(ui.draggable).detach().css({top: 0,left: 0}).appendTo(this);
```
Comment (ulalu) [inline app/assets/javascripts/game_moves.js:19]: what does the {top: 0,left: 0} portion of this code accomplish?

File: app/assets/stylesheets/master.scss [hunk 4]
```diff
@@ -152,11 +152,14 @@ body {
...
+  height: 50px;
+  width: 50px;
+
 }
 
-#board td.present img{ 
+#board td img{ 
   height: auto;
   width: 100%;
 }
```
Comment (ulalu) [inline app/assets/stylesheets/master.scss:165]: good refactor

**Review by ulalu — APPROVED** [review]
Looks great to me, some duplication of the earlier pull request but so long as they're merged in order shouldn't be a problem. Just had the one question for personal edification. Great work!

File: app/assets/javascripts/game_moves.js (hunk 3 above)
Comment (Zjohn64) [reply app/assets/javascripts/game_moves.js:19]: This is the Snap-To-Middle functionality, so it will always center in the square you leave it one

**Pull request merged**